resources/monitoring/charts/alertmanager/README.md

```diff
@@ -1,43 +1,25 @@
 # Alertmanager
 
-This chart contains a part of the configuration related to the Alertmanager.
-
-#### Secret configuration
 
-Alertmanager instances require a Secret resource named with the `alertmanager-{ALERTMANAGER_NAME}` format.
+## Overview 
 
-In Kyma, the name of the Alertmanager is defined by ```name: {{ .Release.Name }}```. The secret is ```name: alertmanager-{{ .Release.Name }}```. The name of the config file is alertmanager.yaml.
+This chart contains a part of the configuration related to the Alertmanager.
 
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  labels:
-    alertmanager: {{ .Release.Name }}
-    app: {{ template "alertmanager.name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
-  name: alertmanager-{{ .Release.Name }}
-data:
-  alertmanager.yaml: |-
-    {{ include "alertmanager.yaml.tpl" . | b64enc }}
-{{- range $key, $val := .Values.templateFiles }}
-  {{ $key }}: {{ $val | b64enc | quote }}
-{{- end }}
-```
+For details on Alertmanager configuration, see [this](https://github.com/kyma-project/kyma/docs/monitoring/03-01-alertmanager.md) document.

```

Comment:
```suggestion
For details on Alertmanager configuration, see [this](https://github.com/kyma-project/kyma/blob/master/docs/monitoring/03-01-alertmanager.md) document.
```

Reply:
but I think we should point to the website, hm?

Reply:
We can but will it work in the `resources` dir?